src/AutoRest.CSharp/Common/Output/Models/RestClientBuilder.cs

```diff
@@ -812,6 +819,28 @@ public void AddRequestContext()
                 _parameters.Add(KnownParameters.RequestContext);
             }
 
+            private void AddContentTypeRequestParameter(RequestParameter requestParameter, ICollection<string> requestMediaTypes)
+            {
+                var name = requestParameter.CSharpName();
+                var description = CreateDescriptionWithMediaTypes(requestParameter, requestMediaTypes);
```

Comment:
The difference between normal content type parameter is the description?

Reply:
The difference between a parameter which is not a content type is the description.